static/index.js

```diff
@@ -44,11 +44,21 @@ function lookup_address(cell, address) {
 }
 
 function renderLaTeX() {
+    // Render LaTeX equations in prose blocks via KaTeX, if available.
+    // Skip rendering if KaTeX is unavailable.
+    if (typeof renderMathInElement == 'undefined') {
+        return;
+    }
     // Render LaTeX equations in prose blocks via KaTeX.
     var proseBlocks = document.querySelectorAll(".prose-block");
     Array.from(proseBlocks).map((proseBlock) =>
         renderMathInElement(proseBlock, katexOptions)
     );
+        var proseBlocks = document.querySelectorAll(".prose-block");
+        Array.from(proseBlocks).map((proseBlock) =>
+            renderMathInElement(proseBlock, katexOptions)
+        );
```

Comment:
What happened here? This code seems to be duplicated?

Reply:
Yes – fixed later by @axch in https://github.com/google-research/dex-lang/commit/02ac25c2ce77e71cb61c67b2a7a889a6c70eeace.